### PR TITLE
Migration from dunfell to rdk-next

### PR DESCRIPTION
**Pull request by Rajakumaran1106**
Signed-off-by: Rajakumaran1106 <rajakumaran.a@ltts.com>

### CODE REVIEW

File: conf/include/rdk-bbmasks-broadband.inc [hunk 1]
```diff
@@ -0,0 +1,5 @@
+BBMASK_dunfell .= "|meta-rdk/recipes-core/packagegroups/packagegroup-rdk-media-common.bb"
```
Comment (manigandanj) [inline conf/include/rdk-bbmasks-broadband.inc:1]: you can add these lines in conf/include/turris-bbmasks-broadband.inc.

File: conf/machine/turris.conf [hunk 2]
```diff
@@ -27,10 +28,12 @@ PREFERRED_PROVIDER_virtual/bootloader ??= "u-boot-turris"
 PREFERRED_VERSION_linux-marvell = "4.14.22"
 PREFERRED_VERSION_linux-libc-headers = "4.14.22"
 PREFERRED_VERSION_xfsprogs = "4.8.0"
+PREFERRED_VERSION_php_dunfell = "7.1.%"
+PREFERRED_VERSION_php-native_dunfell = "7.1.%"
 
 PREFERRED_PROVIDER_hal-wifi = "hal-wifi-turris"
 
-#require conf/include/rdk-bbmasks-broadband.inc
```
Comment (manigandanj) [inline conf/machine/turris.conf:33]: you can remove this line.

File: recipes-bsp/u-boot/files/0001-efi_loader-avoid-make-race-condition.patch [hunk 3]
```diff
@@ -0,0 +1,51 @@
+From 5c2e24a9ed54dfee77d1844a080e998b4affe916 Mon Sep 17 00:00:00 2001
```
Comment (manigandanj) [inline recipes-bsp/u-boot/files/0001-efi_loader-avoid-make-race-condition.patch:1]: No need to add these files again as we are using u-boot-turris

File: recipes-bsp/u-boot/files/MPC8315ERDB-enable-DHCP.patch [hunk 4]
```diff
@@ -0,0 +1,19 @@
+Enabled dhcp client functionality for Yocto reference
```
Comment (manigandanj) [inline recipes-bsp/u-boot/files/MPC8315ERDB-enable-DHCP.patch:1]: No need to add these files again as we are using u-boot-turris

File: recipes-bsp/u-boot/files/armada_38x_defconfig [hunk 5]
```diff
@@ -0,0 +1,242 @@
+CONFIG_SPI_FLASH_WINBOND=y
```
Comment (manigandanj) [inline recipes-bsp/u-boot/files/armada_38x_defconfig:1]: No need to add this file again as we are using u-boot-turris.

File: recipes-bsp/u-boot/files/u-boot-inline-error.patch [hunk 6]
```diff
@@ -0,0 +1,31 @@
+diff --git a/fs/jffs2/mini_inflate.c b/fs/jffs2/mini_inflate.c
```
Comment (manigandanj) [inline recipes-bsp/u-boot/files/u-boot-inline-error.patch:1]: No need to add this file again as we are using u-boot-turris.

File: recipes-bsp/u-boot/u-boot-common_2018.01.inc [hunk 7]
```diff
@@ -0,0 +1,18 @@
+HOMEPAGE = "http://www.denx.de/wiki/U-Boot/WebHome"
```
Comment (manigandanj) [inline recipes-bsp/u-boot/u-boot-common_2018.01.inc:1]: No need to add this file again as we are using u-boot-turris.

File: recipes-bsp/u-boot/u-boot-fw-utils_2018.01.bbappend [hunk 8]
```diff
@@ -0,0 +1,13 @@
+FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
```
Comment (manigandanj) [inline recipes-bsp/u-boot/u-boot-fw-utils_2018.01.bbappend:1]: No need to add this file again as we are using u-boot-fw-utils-turris_2019.07.

File: recipes-bsp/u-boot/u-boot-marvell_2013.01.bbappend [hunk 9]
```diff
@@ -0,0 +1,11 @@
+B = "${S}"
```
Comment (manigandanj) [inline recipes-bsp/u-boot/u-boot-marvell_2013.01.bbappend:1]: No need to add this file again as we are using u-boot-turris.

File: recipes-bsp/u-boot/u-boot-script-marvell_2013.01.bbappend [hunk 10]
```diff
@@ -0,0 +1,13 @@
+FILESEXTRAPATHS_prepend := "${THISDIR}/u-boot-script-marvell/db-88f6820-gp:"
```
Comment (manigandanj) [inline recipes-bsp/u-boot/u-boot-script-marvell_2013.01.bbappend:1]: No need to add this file again as we are using u-boot-script-turris_2019.07.

File: recipes-ccsp/ccsp/ccsp-eth-agent.bbappend [hunk 11]
```diff
@@ -2,7 +2,7 @@ require ccsp_common_turris.inc
 
 EXTRA_OECONF_append  = " --with-ccsp-arch=arm"
 
-LDFLAGS_append =" \
-    -lsyscfg \
-    -lbreakpadwrapper \
-"
+LDFLAGS += " -lbreakpadwrapper \
+             -lsyscfg \
```
Comment (manigandanj) [inline recipes-ccsp/ccsp/ccsp-eth-agent.bbappend:6]: make sure indent is correct here.

File: recipes-common/mesh-agent/mesh-agent.bbappend [hunk 12]
```diff
@@ -9,11 +9,11 @@ patch  -p1 < ${WORKDIR}/enabling_dhcp_lease_resync.patch ${S}/source/MeshAgentSs
 }
 
 do_install_append () {
-	install -D -m 0644 ${S}/systemd_units/meshAgent.service ${D}${systemd_unitdir}/system/meshAgent.service
+       install -D -m 0644 ${S}/systemd_units/meshAgent.service ${D}${systemd_unitdir}/system/meshAgent.service
```
Comment (manigandanj) [inline recipes-common/mesh-agent/mesh-agent.bbappend:12]: make sure indent is uniform.

File: recipes-connectivity/hostapd/files/hostapd-5G.conf [hunk 13]
```diff
@@ -34,9 +34,10 @@ bssid=
 
 #Security parameters
 wpa=2
-wpa_pairwise=CCMP
```
Comment (manigandanj) [inline recipes-connectivity/hostapd/files/hostapd-5G.conf:37]: To fix authentication issue, i made this change. Are you sure that "Authentication" is successful with this change.

File: recipes-connectivity/hostapd/files/hostapd-2G.conf [hunk 14]
```diff
@@ -28,9 +28,10 @@ bssid=
 
 #Security parameters
 wpa=2
-wpa_pairwise=CCMP
```
Comment (manigandanj) [inline recipes-connectivity/hostapd/files/hostapd-2G.conf:31]: To fix authentication issue, i made this change. Are you sure that "Authentication" is successful with this change.

File: recipes-connectivity/opensync/opensync_1.4.0.bbappend [hunk 15]
```diff
@@ -1,5 +1,5 @@
 FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
-
+
```
Comment (manigandanj) [inline recipes-connectivity/opensync/opensync_1.4.0.bbappend:2]: can remove this line.

**Review by manigandanj — CHANGES_REQUESTED** [review]

File: recipes-connectivity/opensync/opensync_1.4.0.bbappend (hunk 15 above)
Comment (Rajakumaran1106) [reply recipes-connectivity/opensync/opensync_1.4.0.bbappend:2]: Noted

File: recipes-connectivity/hostapd/files/hostapd-5G.conf (hunk 13 above)
Comment (Rajakumaran1106) [reply recipes-connectivity/hostapd/files/hostapd-5G.conf:37]: I faced the authentication issue , yes
I will remove it

File: recipes-ccsp/ccsp/ccsp-eth-agent.bbappend (hunk 11 above)
Comment (Rajakumaran1106) [reply recipes-ccsp/ccsp/ccsp-eth-agent.bbappend:6]: Noted

File: recipes-bsp/u-boot/files/0001-efi_loader-avoid-make-race-condition.patch (hunk 3 above)
Comment (Rajakumaran1106) [reply recipes-bsp/u-boot/files/0001-efi_loader-avoid-make-race-condition.patch:1]: I  added this , to provide the compatibility  with older u-boot

File: conf/machine/turris.conf (hunk 2 above)
Comment (Rajakumaran1106) [reply conf/machine/turris.conf:33]: Done

File: conf/include/rdk-bbmasks-broadband.inc (hunk 1 above)
Comment (Rajakumaran1106) [reply conf/include/rdk-bbmasks-broadband.inc:1]: Noted

**Pull request closed without merging**